### ISSUE TITLE
Error on installation

### ISSUE THREAD

**Issue by nWidart** [javiereguiluz/easy-admin-demo#40]
Hello,

I'm getting the following error after the composer install & starting the install script.

```
Fatal error: Uncaught Symfony\Component\Debug\Exception\FatalThrowableError: Type error: Return value of ProxyManager\Configuration::setGeneratorStrategy() must be an instance of ProxyManager\void, none returned in /Users/nicolaswidart/Sites/Tests/easy-admin-demo/vendor/ocramius/proxy-manager/src/ProxyManager/Configuration.php:124
Stack trace:
#0 /Users/nicolaswidart/Sites/Tests/easy-admin-demo/vendor/symfony/symfony/src/Symfony/Bridge/ProxyManager/LazyProxy/Instantiator/RuntimeInstantiator.php(37): ProxyManager\Configuration->setGeneratorStrategy(Object(ProxyManager\GeneratorStrategy\EvaluatingGeneratorStrategy))
#1 /Users/nicolaswidart/Sites/Tests/easy-admin-demo/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php(625): Symfony\Bridge\ProxyManager\LazyProxy\Instantiator\RuntimeInstantiator->__construct()
#2 /Users/nicolaswidart/Sites/Tests/easy-admin-demo/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php(574): Symfony\Component\HttpKernel\Kernel->getContainerBuilder()
#3 /Users/nicolaswid in /Users/nicolaswidart/Sites/Tests/easy-admin-demo/vendor/ocramius/proxy-manager/src/ProxyManager/Configuration.php on line 124
PHP Fatal error:  Uncaught Symfony\Component\Debug\Exception\FatalThrowableError: Type error: Return value of ProxyManager\Configuration::setGeneratorStrategy() must be an instance of ProxyManager\void, none returned in /Users/nicolaswidart/Sites/Tests/easy-admin-demo/vendor/ocramius/proxy-manager/src/ProxyManager/Configuration.php:124
Stack trace:
#0 /Users/nicolaswidart/Sites/Tests/easy-admin-demo/vendor/symfony/symfony/src/Symfony/Bridge/ProxyManager/LazyProxy/Instantiator/RuntimeInstantiator.php(37): ProxyManager\Configuration->setGeneratorStrategy(Object(ProxyManager\GeneratorStrategy\EvaluatingGeneratorStrategy))
#1 /Users/nicolaswidart/Sites/Tests/easy-admin-demo/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php(625): Symfony\Bridge\ProxyManager\LazyProxy\Instantiator\RuntimeInstantiator->__construct()
#2 /Users/nicolaswidart/Sites/Tests/easy-admin-demo/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php(574): Symfony\Component\HttpKernel\Kernel->getContainerBuilder()
#3 /Users/nicolaswid in /Users/nicolaswidart/Sites/Tests/easy-admin-demo/vendor/ocramius/proxy-manager/src/ProxyManager/Configuration.php on line 124
Script Sensio\Bundle\DistributionBundle\Composer\ScriptHandler::clearCache handling the post-install-cmd event terminated with an exception


  [RuntimeException]
  An error occurred when executing the "'cache:clear --no-warmup'" command:
  Fatal error: Uncaught Symfony\Component\Debug\Exception\FatalThrowableError: Type error: Return value of ProxyManager\Conf
  iguration::setGeneratorStrategy() must be an instance of ProxyManager\void, none returned in /Users/nicolaswidart/Sites/Te
  sts/easy-admin-demo/vendor/ocramius/proxy-manager/src/ProxyManager/Configuration.php:124
  Stack trace:
  #0 /Users/nicolaswidart/Sites/Tests/easy-admin-demo/vendor/symfony/symfony/src/Symfony/Bridge/ProxyManager/LazyProxy/Insta
  ntiator/RuntimeInstantiator.php(37): ProxyManager\Configuration->setGeneratorStrategy(Object(ProxyManager\GeneratorStrateg
  y\EvaluatingGeneratorStrategy))
  #1 /Users/nicolaswidart/Sites/Tests/easy-admin-demo/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php(625
  ): Symfony\Bridge\ProxyManager\LazyProxy\Instantiator\RuntimeInstantiator->__construct()
  #2 /Users/nicolaswidart/Sites/Tests/easy-admin-demo/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php(574
  ): Symfony\Component\HttpKernel\Kernel->getContainerBuilder()
  #3 /Users/nicolaswid in /Users/nicolaswidart/Sites/Tests/easy-admin-demo/vendor/ocramius/proxy-manager/src/ProxyManager/Co
  nfiguration.php on line 124
  PHP Fatal error:  Uncaught Symfony\Component\Debug\Exception\FatalThrowableError: Type error: Return value of ProxyManager
  \Configuration::setGeneratorStrategy() must be an instance of ProxyManager\void, none returned in /Users/nicolaswidart/Sit
  es/Tests/easy-admin-demo/vendor/ocramius/proxy-manager/src/ProxyManager/Configuration.php:124
  Stack trace:
  #0 /Users/nicolaswidart/Sites/Tests/easy-admin-demo/vendor/symfony/symfony/src/Symfony/Bridge/ProxyManager/LazyProxy/Insta
  ntiator/RuntimeInstantiator.php(37): ProxyManager\Configuration->setGeneratorStrategy(Object(ProxyManager\GeneratorStrateg
  y\EvaluatingGeneratorStrategy))
  #1 /Users/nicolaswidart/Sites/Tests/easy-admin-demo/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php(625
  ): Symfony\Bridge\ProxyManager\LazyProxy\Instantiator\RuntimeInstantiator->__construct()
  #2 /Users/nicolaswidart/Sites/Tests/easy-admin-demo/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php(574
  ): Symfony\Component\HttpKernel\Kernel->getContainerBuilder()
  #3 /Users/nicolaswid in /Users/nicolaswidart/Sites/Tests/easy-admin-demo/vendor/ocramius/proxy-manager/src/ProxyManager/Co
  nfiguration.php on line 124
  .

```

Thanks,

**Comment by jschouwstra** [comment]
+1

**Comment by javiereguiluz** [comment]
I've updated dependencies in #49 and defined the PHP version to use in #48:

```json
"config": {
        "platform": {
            "php": "5.5.9"
        },
    },
```

The installation works now as expected. I'm sorry it took me so long to fix this issue for people not using PHP 7.1. Cheers!
